x/attribute/keeper/keeper.go

```diff
@@ -297,6 +302,12 @@ func (k Keeper) DeleteAttribute(ctx sdk.Context, addr string, name string, value
 	return nil
 }
 
+func (k Keeper) GetDefaultAttributeExpiration(ctx sdk.Context, attr types.Attribute) string {
+	// use default duration param value from expiration module now,
+	// but may be calculated in the future based on attribute info.
+	return exptypes.DefaultDuration
+}
+
```

Comment:
Since there's nothing looked up here, this function doesn't need to take in any arguments.

That being said, the expiration params have a `duration` entry, but it's not being used here. that `exptypes.DefaultDuration` is code defined and never updated, so it's not going to match what the params have.